quinn-proto/src/connection/streams/mod.rs

```diff
@@ -79,6 +80,17 @@ impl<'a> Streams<'a> {
     pub fn send_streams(&self) -> usize {
         self.state.send_streams
     }
+
+    /// The number of remotely initiated open streams of a certain directionality.
+    ///
+    /// Includes remotely initiated streams, which have not been accepted via [`accept`](Self::accept).
+    /// These streams count against the respective concurrency limit reported by
+    /// [`Connection::max_concurrent_streams`](super::Connection::max_concurrent_streams).
+    pub fn remote_open_streams(&self, dir: Dir) -> u64 {
+        self.state.next_remote[dir as usize]
+            - (self.state.max_remote[dir as usize]
+                - self.state.allocated_remote_count[dir as usize])
```

Comment:
It took me a few minutes to understand this math (number of streams ever opened minus number of streams ever closed), so it could probably use a comment.

Reply:
Will do, if we keep it at all.

Reply:
Changed to this:
```Rust
// total opened - total closed = total opened - ( total permitted - total permitted unclosed )
        self.state.next_remote[dir as usize]
            - (self.state.max_remote[dir as usize]
                - self.state.allocated_remote_count[dir as usize])
```